integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -44,22 +44,27 @@
                                 "spark.sql.legacy.parquet.datetimeRebaseModeInRead": "CORRECTED",
                                 "spark.sql.legacy.parquet.int96RebaseModeInRead": "CORRECTED"})
 
+def get_writer_with_deletion_vector_property_set(writer, enable_deletion_vectors):
+    return writer.option("delta.enableDeletionVectors", str(enable_deletion_vectors).lower())
+
 def get_last_operation_metrics(path):
     from delta.tables import DeltaTable
     return with_cpu_session(lambda spark: DeltaTable.forPath(spark, path)\
                             .history(1)\
                             .selectExpr("operationMetrics")\
                             .head()[0])
 
-def _create_table(spark, path, schema, partitioned_by=None):
+def _create_table(spark, path, schema, partitioned_by=None, enable_deletion_vectors=False):
     q = f"CREATE TABLE delta.`{path}` ({schema}) USING DELTA"
     if partitioned_by:
         q += f" PARTITIONED BY ({partitioned_by})"
+
+    q += " TBLPROPERTIES ('delta.enableDeletionVectors' = {})".format(str(enable_deletion_vectors).lower())
     spark.sql(q)
 
-def _create_cpu_gpu_tables(spark, path, schema, partitioned_by=None):
-    _create_table(spark, path + "/CPU", schema, partitioned_by)
-    _create_table(spark, path + "/GPU", schema, partitioned_by)
+def _create_cpu_gpu_tables(spark, path, schema, partitioned_by=None, enable_deletion_vectors=False):
```

Comment:
Same question here. I get it if we just didn't want to change the order of params, but we have full control over that here.